pkg/testcase/secretsencrypt.go

```diff
@@ -22,65 +24,72 @@ func TestSecretsEncryption() {
 	errSecret := shared.CreateSecret("secret1", "default")
 	Expect(errSecret).NotTo(HaveOccurred(), "error creating secret")
 
-	shared.LogLevel("info", "TEST: 'CLASSIC' Secrets Encryption method")
-
 	index := len(nodes) - 1
 	cpIp := nodes[index].ExternalIP
```

Comment:
Will it be always true that the last index in nodes array will be the CP node?

Reply:
yes. we either have a setup with etcd + cp nodes. or 1 etcd 2 cp, 3 etcd, 2cp etc. we get the nodes by roles - etcd first and next cp. we wont have a setup without a cp node. and it will always be there on the last node - since we get them by roles in that order.